### ISSUE TITLE
Mii interactive menu being triggered by "source"d scripts. 

### ISSUE THREAD

**Issue by mboisson** [codeandkey/mii#17]
Mii's interactive menu is being triggered by scripts that are sourced: 

```
$ cat t.sh
#!/bin/bash
blastn
$ ./t.sh 
./t.sh: line 2: blastn: command not found
``` 
The above is good. The below is bad: 
``` 
$ . t.sh
[mii] Please select a module to run blastn:
       MODULE         PARENT(S)
    1  blast+/2.11.0  StdEnv/2020 gcc/9.3.0
    2  blast+/2.10.1  StdEnv/2020 gcc/9.3.0
``` 

Solutions to check for this are detailed here: 
https://stackoverflow.com/questions/2683279/how-to-detect-if-a-script-is-being-sourced

I will try to get a PR in next week if I find the time.

**Comment by codeandkey** [comment]
Thank you for the bug report! Should be implemented in #18, let me know if this is working for you.

**Comment by mboisson** [comment]
Yes, that seems to be working for me.
